pieces.py

```diff
@@ -3,24 +3,65 @@ def __init__(self, color: str, i: int, j: int):
         self.color = color
         self.position = (i, j)
         self.firstMove = True
+    def valid_moves(self, board): #Returns a list of valid coordinates for respective piece
```

Comment:
```suggestion
    def valid_moves(self, board):
    """Returns a list of valid coordinates for respective piece."""
```

please read this article on how to write docstrings: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html